src/foam/nanos/boot/Boot.java

```diff
@@ -13,23 +13,25 @@
 import foam.dao.java.JDAO;
 import foam.dao.ProxyDAO;
 import foam.nanos.auth.Group;
-import foam.nanos.auth.Permission;
 import foam.nanos.auth.User;
 import foam.nanos.logger.Logger;
 import foam.nanos.logger.ProxyLogger;
 import foam.nanos.logger.StdoutLogger;
 import foam.nanos.script.Script;
 import foam.nanos.session.Session;
-import java.util.ArrayList;
-import java.util.List;
+
+import java.util.*;
+
 import static foam.mlang.MLang.EQ;
 
 public class Boot {
   // Context key used to store the top-level root context in the context.
   public final static String ROOT = "_ROOT_";
+  private final static Set<String> RELOADED_SERVICES = new HashSet<>();
```

Comment:
What is this Set used for?

Reply:
It's for storing the reloaded service name for use in reloading web agents and nano services.